qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -102,27 +127,20 @@ def test_mkdir_with_07000_octal_mode(self):
         """
         Test that mkdir fails with octal mode greater than 0777
         """
-        o = self.get_cephfs_shell_cmd_output("mkdir -m 07000 d2")
-        log.info("cephfs-shell output:\n{}".format(o))
+        self.negtest_cephfs_shell_cmd("mkdir -m 07000 d2")
 
-        # mkdir d2 should fail
         try:
-            o = self.mount_a.stat('d2')
-            log.info("mount_a output:\n{}".format(o))
-        except:
+            self.mount_a.stat('d2')
+        except CommandFailedError:
             pass
 
     def test_mkdir_with_negative_octal_mode(self):
         """
         Test that mkdir fails with negative octal mode
         """
-        o = self.get_cephfs_shell_cmd_output("mkdir -m -0755 d3")
-        log.info("cephfs-shell output:\n{}".format(o))
-
-        # mkdir d3 should fail
+        self.negtest_cephfs_shell_cmd("mkdir -m -0755 d3")
         try:
-            o = self.mount_a.stat('d3')
-            log.info("mount_a output:\n{}".format(o))
+            self.mount_a.stat('d3')
         except:
```

Comment:
```suggestion
        except CommandFailedError:
```

Reply:
It is missing in other tests of mkdir.

Reply:
We can use a new common method for testing of non existing directory in mkdir.

Reply:
Right now the intention is to just update tests based on the changes in cephfs-shell and not improve and clean up them.

Reply:
Since you are making changes to the code, it is fine if we fix it too.